src/app/partnerzy/page.tsx

```diff
@@ -1,5 +1,70 @@
-import { ComingSoon } from "@/components/ComingSoon/ComingSoon";
+import Container from "@/components/shared/Container";
+import { Navigation } from "./_components/Navigation";
+import { Paragraph } from "./_components/Paragraph";
+import { BulletList } from "./_components/BulletList/BulletList";
+import { Partnership } from "./_components/Partnership";
+import { Thread } from "./_components/Thread";
+import { OurPartners } from "./_components/OurPartners";
+import { ThankYou } from "./_components/ThankYou";
+import { AK_DATA } from "@/data/akData";
+
+// Todo: Add meta for partners for SEO? Carousel is client-side.
```

Comment:
Not sure about this. We can discuss it

Reply:
@NacomiTagiera  what about this? The Carousel is a client component. And it will contain all Partners data.
- Should we include Partners data in some metatags (og)?